VideoWeb/VideoWeb/ClientApp/src/app/shared/pipes/ngx-date.pipe.ts

```diff
@@ -0,0 +1,16 @@
+import { DatePipe } from '@angular/common';
+import { Pipe, PipeTransform } from '@angular/core';
+import { TranslateService } from '@ngx-translate/core';
+
+@Pipe({
+    name: 'translateDate',
+    pure: false
+})
+export class NgxDatePipe implements PipeTransform {
+    constructor(private translateService: TranslateService) {}
+
+    public transform(value: any, pattern: string): any {
```

Comment:
I think pattern should be format to match DatePipe

https://angular.io/api/common/DatePipe

Reply:
As mentioned elsewhere I'm not sure if Enum is the way to go, looking into that now. But if we do can this be changed from a string to that Enum?